vignettes/design-principles.Rmd

```diff
@@ -29,6 +29,34 @@ This package should work with plots provided by:
 - ggplot2
 - ggraph
 
+## Design decisions
+
+### Alignment with Epiverse-TRACE web theme
+
+Epiverse-TRACE already has a clear visual identity for its web resources, via the [pkgdown custom theme](https://epiverse-trace.github.io/epiversetheme), the [varnish fork to customize the look of online training materials](https://epiverse-trace.github.io/varnish), and the [custom theme for the quarto website](https://github.com/epiverse-trace/epiverse-trace.github.io/blob/0d70e09531c422648e0a1d46f7ca64c45def78a2/custom.scss).
+
+The custom ggplot2 theme and scale provided here should roughly align with this existing visual identity.
+
+In practice, this means:
+
+- The default font should be [`DM Sans`](https://fonts.google.com/specimen/DM+Sans), or [`Roboto Mono`](https://fonts.google.com/specimen/Roboto+Mono) for code.
+- The colour scale should use one or more of the Epiverse-TRACE brand colours:
+  - [Indigo](https://color-hex.org/color/071e2d)
+  - [Deep Coral](https://color-hex.org/color/f04a4c)
+  - [Ether Blue](https://color-hex.org/color/106ba0)
+  - [Aqua](https://color-hex.org/color/10bed2)
+  - [Olive](https://color-hex.org/color/aec800)
+  - [Lime](https://color-hex.org/color/deff00)
+  - [Cool Gray](https://color-hex.org/color/88888)
+  - [Warm Gray](https://color-hex.org/color/ebe6e0)
```

Comment:
A number of talented developers have put a good deal of thought into creating colour palettes for scientific data while taking into account perceptual uniformity, suitability for colour blindness types, and greyscale print-friendliness, and providing these as R packages that integrate well with {ggplot2} (such as [{scico}](https://cran.r-project.org/package=scico)).

We would ideally choose 1-3 palettes from among these, perhaps some options which resemble the Epiverse colours, and advise all developers to use them in vignettes etc. (although I'm open to the idea that some Epiverse colour combinations could work for categorical data).

Reply:
I agree with @pratikunterwegs here. However, we can discuss about the palettes mentioned in the Epiverse Brand book, which include diverging and sequential color palettes (Image attached for reference). Also, we could add Denis (she supports the LAC graphic design) to this discussion and ask for her help considering the usage of palettes that might already exist but include the Epiverse brand colors.

![image](https://github.com/epiverse-trace/tracetheme/assets/56242573/90188096-fab2-46ad-ab32-a37ada4f4119)

Reply:
Thanks both, I agree with you!

The goal is not to start from scratch and reinvent a worse solution than what others have spent months or years on. However, in many cases, these researchers and engineers have published their methods or algorithms to generate a perceptually uniform colour scale ([blog post and link to code for viridis](https://bids.github.io/colormap/), [video for viridis](https://www.youtube.com/watch?v=xAoljeRJ3lU)).

The idea is to see if one or more of the Epiverse brand colours can be used as seed in these algorithms to generate a perceptually uniform scale, while having an Epiverse feel.

If it turns out to not be possible, I agree we can use existing palettes that match well with the brand palette.

Denis is definitely welcome in this task and finish working group.

Reply:
At yesterday's meeting we decided to attempt creating 3 colour scales (continuous, diverging, and categorical) from the Epiverse colours, and to choose existing scales to recommend if we are not able to create suitable scales. Our work is expected within a week (see linked doc on Slack channel) - I will look into this later this week.

Reply:
We can talk about it later this week over slack to choose the palettes and define the scales.

Reply:
Sure, can coordinate over Slack